packages/ckeditor5-list/ckeditor5-metadata.json

```diff
@@ -87,7 +87,8 @@
 					],
 					"attributes": [
 						"start",
-						"reversed"
+						"reversed",
+						"type"
```

Comment:
Are you sure this is OK? Only the `DocumentListProperties` supports the `type` attribute. `ListProperties` is incapable of returning `type` in the data pipeline.